src/gui/symbology/qgsgraduatedsymbolrendererwidget.cpp

```diff
@@ -906,6 +910,8 @@ void QgsGraduatedSymbolRendererWidget::symmetryPointEditingFinished( )
 
 void QgsGraduatedSymbolRendererWidget::classifyGraduated()
 {
+
+  QApplication::setOverrideCursor( Qt::WaitCursor );
```

Comment:
There's a scoped class in GUI which handles this nicely

Reply:
(QgsTemporaryCursorOverride)

Reply:
Thanks! I was pretty sure that there was a scoped helper but I didn't search hard enough.
Do you think we could collect all this little helpers in a dedicated doxy section?